custodian/qchem/utils.py

```diff
@@ -0,0 +1,34 @@
+# coding: utf-8
+
+"""
+This module contains utility functions that are useful for Q-Chem jobs.
+"""
+
+import math
+
+import numpy as np
+
+
+def perturb_coordinates(old_coords, negative_freq_vecs, molecule_perturb_scale, reversed_direction):
+    """
+    Perturbs a structure along the imaginary mode vibrational frequency vectors
```

Comment:
Pls add documentaion on variables. This applies to all methods.

Reply:
It would also help to add functional annotations since a lot of this is new code.